client/src/partials/patient_invoice/registry/modalCreditNote.html

```diff
@@ -0,0 +1,70 @@
+<form  name="ModalForm" ng-submit="ModalCtrl.submit(ModalForm)" bh-form-defaults novalidate>
+  <div class="modal-header">
+    <div class="bhima-title">
+      <ol class="headercrumb">
+        <li class="static">{{ "FORM.INFOS.INVOICES" | translate }}</li>
+        <li class="title">{{ "FORM.INFOS.CREDIT_NOTE" | translate }}</li>
+      </ol>
+    </div>
+  </div>
+
+  <div class="modal-body" style="overflow: auto; max-height: 2000px;">
```

Comment:
That's a very large max height!

In the future, it would be good to limit a modal's height to a part of the page height.  You can do it with the `vh` CSS unit.  [Here is the documentation](https://developer.mozilla.org/en/docs/Web/CSS/length#Viewport-percentage_lengths).  Like this:
```html
<!-- this element will never be taller than 95% of the screen height!  Yay! -->
<element style="max-height: 95vh;">
</element>
```